graphql/resolvers.go

```diff
@@ -1070,6 +1088,25 @@ func (r *queryResolver) MyHosts(ctx context.Context) ([]*restModel.APIHost, erro
 	return apiHosts, nil
 }
 
+func (r *queryResolver) Permissions(ctx context.Context, userID string) (*Permissions, error) {
```

Comment:
At this point, it seems like the definite majority of use cases for this resolver will relate to the current user accessed via context; it would be more cumbersome to get the user ID on the front end instead of looking it up in the query. My inclination is that we should at least offer a version of this resolver that does not require a user ID, but interested in other opinions!